app/src/main/java/com/duckduckgo/app/fire/AutomaticDataClearer.kt

```diff
@@ -61,7 +67,8 @@ class AutomaticDataClearer @Inject constructor(
     private val settingsDataStore: SettingsDataStore,
     private val clearDataAction: ClearDataAction,
     private val dataClearerTimeKeeper: BackgroundTimeKeeper,
-    private val dataClearerForegroundAppRestartPixel: DataClearerForegroundAppRestartPixel
+    private val dataClearerForegroundAppRestartPixel: DataClearerForegroundAppRestartPixel,
+    private val dispatchers: DispatcherProvider
```

Comment:
is it possible to use `dispatchers.main` also to initialize `coroutineContext` inside this class?